app/views/announcement_mailer/multi_chan.html.slim

```diff
@@ -40,8 +40,8 @@ p= mail_to(Rails.application.secrets[:support_email])
 p Please note our support staff is quite small, however we will do our best to help you.
 
 
-p We hope you enjoy this new functionality as we strive to make Brave Payments efficient and easy to use. Thanks for your patience, and thank you for supporting the Basic Attention Token.
+p We hope you enjoy this new functionality as we strive to make Brave Rewards efficient and easy to use. Thanks for your patience, and thank you for supporting the Basic Attention Token.
 
 div Ayumi Yu
-div Technical Lead, Brave Payments
+div Technical Lead, Brave Rewards
```

Comment:
we can remove this file and associated mailer methods

Reply:
Removed 😄 
https://github.com/brave-intl/publishers/pull/1519/commits/0a39725762f5c8df56667dada109933ea7d79f48